content/strategy-goals/strategy/cloud/iam-admin/index.md

```diff
@@ -1,5 +1,47 @@
 # IAM and Admin Experience Team
 
-## Current and Future Goals
+This page outlines the short-term strategy for the IAM and Admin Experience Team. You can find the overview of Sourcegraph Product and Engineering FY23 Q2 focus [here](../../../../departments/product-engineering/strategy-goals/index.md#fy23-q2-focus)
 
-This team is newly formed and is currently defining goals.
+## Q2 Objective and Key Results
+
+### Problem Area: Align on Team Vision
```

Comment:
qq: These are sub sections of "Objective and Key Results" but why is they are named as "Problem Area:" instead of "Objective:"?

Reply:
@unknwon - Good question and callout. I can update the headline to match. 

There are a few reasons: 
- Since we are fairly new, the various areas are at different levels of scoping. So we felt defining as problem areas was a good normalizer for areas we do not have any outcomes defined yet. 
- It brings the user voice problem into the planning process so we make sure our plans for the quarter are tied to user problems
- It lets our team be in the driver seat of defining the best solution, instead of embedding a solution in results.

Reply:
@unknwon - I'm going to merge this and we can update if you feel we should adjust the heading.

Reply:
@ryphil thanks, no need to update the headings as was literally a question :)

Though just speaking on the matter, "solving problem area XYZ" could just be our objectives, if that makes sense to be an "objective" at all.